src/app/components/shared/project-list/project-list.component.html

```diff
@@ -1,46 +1,47 @@
 <div class="card-body">
-  <div class="scroll">
-    <div class="accordion" id="accordionProject">
-      <div *ngIf="projects?.length > 0; else notShow">
-        <div class="card" *ngFor="let project of projects; let rowIndex = index">
-          <div class="card-header">
-            <h2 class="mb-0">
-              <a type="button" data-toggle="collapse" [attr.data-target]="'#row'+rowIndex" [attr.aria-controls]="'row'+rowIndex">
+    <h1 class="card-title">Project List</h1>
+    <hr>
+    <app-search-project (changeFilterProject)="filterProjects = $event"></app-search-project>
+    <hr>
+    <div class="scroll">
+        <div class="accordion" id="accordionProject">
+            <div *ngIf="projects?.length > 0; else notShow">
+                <div class="card" *ngFor="let project of projects | filterProject:filterProjects; let rowIndex = index">
+                    <div class="card-header">
+                        <h2 class="mb-0">
+                            <a type="button" data-toggle="collapse" [attr.data-target]="'#row'+rowIndex" [attr.aria-controls]="'row'+rowIndex">
```

Comment:
Please, fix the indentation, remember that we are using an indentation based in 2 spaces.

Reply:
done